pkg/api/interface.go

```diff
@@ -56,6 +56,18 @@ type Interface interface {
 	GetPackage(*fastly.GetPackageInput) (*fastly.Package, error)
 	UpdatePackage(*fastly.UpdatePackageInput) (*fastly.Package, error)
 
+	CreateDictionary(*fastly.CreateDictionaryInput) (*fastly.Dictionary, error)
+	GetDictionary(*fastly.GetDictionaryInput) (*fastly.Dictionary, error)
+	DeleteDictionary(*fastly.DeleteDictionaryInput) error
+
+	ListDictionaryItems(*fastly.ListDictionaryItemsInput) ([]*fastly.DictionaryItem, error)
+	GetDictionaryItem(*fastly.GetDictionaryItemInput) (*fastly.DictionaryItem, error)
+	CreateDictionaryItem(*fastly.CreateDictionaryItemInput) (*fastly.DictionaryItem, error)
+	UpdateDictionaryItem(*fastly.UpdateDictionaryItemInput) (*fastly.DictionaryItem, error)
```

Comment:
For the 'Dictionary item' section can you also add `UpdateDictionaryItems` using the Go-Fastly function [`BatchModifyDictionaryItems `](https://github.com/fastly/go-fastly/blob/master/fastly/dictionary_item.go#L205) which maps to the APIs [bulk update](https://developer.fastly.com/reference/api/dictionaries/dictionary-item/#bulk-update-dictionary-item).

Reply:
Do you think it should be it's own command?
```
  dictionaryitems update --dictionary-id=DICTIONARY-ID --file=FILE
```

 or a variation of the existing one?
```
  dictionaryitem update --dictionary-id=DICTIONARY-ID --file=FILE
```

Reply:
Semantically, the plural makes sense but the API section is still `dictionary item` even if the endpoint is `/service/service_id/dictionary/dictionary_id/items`. My vote is that it would be excessive to create a new top-level command.

Reply:
I agree that it would be excessive to create a new top-level command for bulk updates and wouldn't follow our API. And like the idea of a `--file` flag to pass a json object 👍

Reply:
Good shout. I'm happy with this suggestion 👍🏻